scopus-import/src/main/java/no/sikt/nva/scopus/conversion/files/ScopusFileConverter.java

```diff
@@ -458,21 +360,94 @@ private ScopusFile saveFile(ScopusFile scopusFile) {
         }
     }
 
-    private HttpResponse<InputStream> fetchResponseAsInputStream(URI uri) {
-        var response = attempt(() -> httpClient.send(constructRequest(uri), BodyHandlers.ofInputStream()))
-                           .orElseThrow();
-        if (isSuccess(response)) {
-            return response;
-        } else {
-            throw new RuntimeException(FETCH_FILE_ERROR_MESSAGE);
-        }
-    }
-
-    private List<UpwOaLocationType> getLocations(DocTp docTp) {
-        return Optional.ofNullable(docTp.getMeta().getOpenAccess())
-                   .map(OpenAccessType::getUpwOpenAccess)
-                   .map(UpwOpenAccessType::getUpwOaLocations)
-                   .map(UpwOaLocationsType::getUpwOaLocation)
-                   .orElse(List.of());
-    }
+    // The functionality to fetch files from XML is currently disabled because the XML data lacks
+    // license information for the files. This code will remain commented out until a decision
+    // is made regarding how to handle files without license information.
+    // Once a solution is implemented, this code can be revisited and potentially re-enabled.
+
+    //    private Optional<List<AssociatedArtifact>> fetchFilesFromXml(DocTp docTp) {
```

Comment:
Do not like commenting it out that way, but I think this is something we very likely will take in use once PO has decided what files we want to import from xml.

Reply:
You can create a PR to reinstate it, but remove it here